packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -82,19 +112,45 @@ const formatAddress = (appealCaseData) => {
 	return addressComponents.filter(Boolean).join(', ');
 };
 
-const formatDecision = (decision) => {
-	switch (decision) {
-		case 'allowed':
-			return 'ALLOWED';
-		case 'dismissed':
-			return 'DISMISSED';
-		case 'split decision':
-			return 'ALLOWED IN PART';
-		default:
-			return decision;
+/**
+ * @param {AppealSubmission} appealSubmission
+ * @returns {string}
+ */
+const formatAppealSubmissionAddress = (appealSubmission) => {
+	if (!appealSubmission.appeal?.appealSiteSection?.siteAddress) {
+		return '';
 	}
+	const address = appealSubmission.appeal?.appealSiteSection?.siteAddress;
+
+	const addressComponents = [
+		address.addressLine1,
+		address.addressLine2,
+		address.town,
+		address.postcode
+	];
```

Comment:
<img width="638" alt="Screenshot 2024-01-09 at 11 08 39" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/acfe3cbb-1164-476f-90e0-1e76a226f6f9">

even with all that optional chaining you might still get in trouble here

Reply:
I think with the if on line 120, this is safe and we probably can remove the optional chaining?

Reply:
oh yeah agreed, I'll resolve this one, I saw you've made this comment already elsewhere